src/content/article/over-a-year-late-a-dollar-short-more-than-two-decades-of-kodi/index.md

```diff
@@ -0,0 +1,87 @@
+
+![XBMC Home Screen, 2009](/images/blog/xbmc-2009-1080p.webp "Image of the 2009 XBMC home screen and menu")
+
+There are also countless other contributors - to the code, artwork, forum, and beyond - who absolutely have to be acknowledged. Today's Kodi wouldn't be what it is without the add-on ecosystem, translators, community moderators, volunteer artists, system admins, database admins, and many more.
+
+And while devs have came and gone, the project has lived on, continuing to expand to every major platform it possibly can, even to a couple of smart TV ones now. Highly unusual - maybe unique - as an open source application that started on a Microsoft product before spreading further to so many other, disparate platforms, Kodi has now even come full circle to be back in the [Xbox Store](https://apps.microsoft.com/detail/9NBLGGH4T892?hl=en-US&gl=US) (although, somewhat ironically, Microsoft platforms have become increasingly challenging for us because we lack current developer knowledge). We still try to bring all the functionality we can to every platform we can: you've got a weird mod file, from your demoscene days? Still got a pile of FLVs? We got you. What about wanting to stream Widevine-enabled Internet content with an open source project? Yeah, we see you and we care.
+
+**So, Here We Are**
+
+One of the parts that some don’t realize is since our initial development, we’ve always been a monolithic application, meaning the entire codebase must be ported to the desired platform. I’ve said we’re the ‘largest open source multimedia project’ for years, half jokingly, but I mean this by literal lines of code that takes to run it.
+
+![XBMC Home Screen, 2012](/images/blog/xbmc-2012-1080p.webp "Image of the 2012 XBMC home screen and menu")
+
+And we’ve stayed volunteer. As a consumer-oriented open source project, no company has ever sponsored more than a port or a feature; indeed, since the nonprofit foundation behind it isn’t setup to be engaged, at worst companies have simply given some money to an individual to work on a feature. Even this is definitely not the norm: 99.9% of Kodi has been built out of passion and love for multimedia and the desire to have an open source platform for which people can decide the look and feel and customize to their liking, in both hardware and software terms. 
```

Comment:
```suggestion
And we’ve stayed volunteer. As a consumer-oriented open-source project, no company has ever sponsored more than a port or a feature; indeed, since the nonprofit foundation behind it isn’t setup to be engaged, at worst companies have simply given some money to an individual to work on a feature. Even this is definitely not the norm: 99.9% of Kodi has been built out of passion and love for multimedia and the desire to have an open-source platform for which people can decide the look and feel and customize to their liking, in both hardware and software terms. 
```

Reply:
There's a theme emerging here :)

Both done.